src/scripts/services/user-service.ts

```diff
@@ -196,14 +196,17 @@ export class OktaUserService {
         )}].`
     );
 
-  readonly activateUser = (userId: string): TE.TaskEither<string, User> =>
+  readonly activateUser = (
```

Comment:
**Issue:** document this method please.

Reply:
added in commit: 6b672568455dca1ecc7b7d13d462bde70475aa8b

Reply:
This method needs a description. When you are next in the office, grab the green elements of java style book to read its section on comments.